src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx

```diff
@@ -199,11 +200,8 @@ const SendFunds = ({
           })
 
           const handleScan = (value, closeQrModal) => {
-            let scannedAddress = value
+            const scannedAddress = getAddressWithoutNetworkPrefix(value)
```

Comment:
`SendFunds` is just one of the many components where a QR code is accepted. Did you check other inputs?

Reply:
Yes, I checked the other modals as well. It was failing in the "Send Funds" due to an extra validation that did not accepted the hash prefixed.